wordpress/wp-content/plugins/cds-base/classes/Modules/ListManager/app/src/components/ListForm.tsx

```diff
@@ -32,100 +32,104 @@ export const ListForm = ({ handler, formData = {}, serverErrors = [] }: { handle
-                            {errors.unsubscribe_phone_template_id && <span className="validation-error">{errors.unsubscribe_phone_template_id?.message}</span>}
-                            <input style={textWidth} type="text" {...register("unsubscribe_phone_template_id")} />
-                        </div>
-                    </td>
-                </tr>
-                <tr>
-                    <th><label className="gc-label required" htmlFor="subscribe_redirect_url">{__("Subscribe redirect url", "cds-snc")}</label></th>
-                    <td>
-                        <div className={errors.subscribe_redirect_url ? "error-wrapper" : ""}>
-                            {errors.subscribe_redirect_url && <span className="validation-error">{errors.subscribe_redirect_url?.message}</span>}
-                            <input style={textWidth} type="text" {...register("subscribe_redirect_url")} />
-                        </div>
-                    </td>
-                </tr>
-                <tr>
-                    <th><label className="required" htmlFor="unsubscribe_redirect_url">{__("Unsubscribe redirect url", "cds-snc")}</label></th>
-                    <td>
-                        <div className={errors.unsubscribe_redirect_url ? "error-wrapper" : ""}>
-                            {errors.unsubscribe_redirect_url && <span className="validation-error">{errors.unsubscribe_redirect_url?.message}</span>}
-                            <input style={textWidth} type="text" {...register("unsubscribe_redirect_url")} />
-                        </div>
-                    </td>
-                </tr>
-                <tr>
-                    <th><label className="required" htmlFor="confirm_redirect_url">{__("Confirm redirect url", "cds-snc")}</label></th>
-                    <td>
-                        <div className={errors.confirm_redirect_url ? "error-wrapper" : ""}>
-                            {errors.confirm_redirect_url && <span className="validation-error">{errors.confirm_redirect_url?.message}</span>}
-                            <input style={textWidth} type="text" {...register("confirm_redirect_url")} />
-                        </div>
-                    </td>
-                </tr>
-                <tr>
-                    <th></th>
-                    <td><input className="button button-primary" type="submit" /></td>
-                </tr>
+                <thead>
+                    <tr>
+                        <th><label className="required" htmlFor="name"><Asterisk />{__("Name", "cds-snc")}</label></th>
+                        <td>
+                            <div className={errors.name ? "error-wrapper" : ""}>
+                                {errors.name && <span className="validation-error">{errors.name?.message || __("Name is required", "cds-snc")}</span>}
+                                <input style={textWidth} type="text" {...register("name", { required: true })} />
```

Comment:
These inputs need an `id` as well, otherwise the form label isn't associated with them.